### DISCUSSION TITLE
Jupyter notebook (VS Code, Mac) PydanticImportErrors...

### DISCUSSION THREAD

**Discussion by nickgreengithub** [Data-Centric-AI-Community/fg-data-profiling#1465] [General]
**Steps**

I've followed the install instructions multiple times and I'm stuck before I even start. Google Collab also doesn't recognise ydata_profiling. Any help would be appreciated. 

pip3 install ydata-profiling  (mac)

... then I tried running this .ipynb in vs code..

import numpy as np
import pandas as pd
from ydata_profiling import ProfileReport

df = pd.DataFrame(np.random.rand(100, 5), columns=["a", "b", "c", "d", "e"])

**Error**

I'm getting PydanticImportError against:

----> 3 from pandas_profiling import ProfileReport

File [/opt/homebrew/lib/python3.11/site-packages/pandas_profiling/__init__.py:8](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/lib/python3.11/site-packages/pandas_profiling/__init__.py:8)
      5 import importlib.util
      6 from warnings import warn
----> 8 from ydata_profiling.compare_reports import compare

File [/opt/homebrew/lib/python3.11/site-packages/ydata_profiling/__init__.py:7](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/lib/python3.11/site-packages/ydata_profiling/__init__.py:7)
      1 """Main module of ydata-profiling.
      2 
      3 .. include:: ../../README.md
      4 """
      5 import importlib.util
----> 7 from ydata_profiling.compare_reports import compare
      8 from ydata_profiling.controller import pandas_decorator
      9 from ydata_profiling.profile_report import ProfileReport
...
    295     raise PydanticImportError(f'`{import_path}` has been removed in V2.')

PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.3/migration/#basesettings-has-moved-to-pydantic-settings for more details.

**Comment by mnd1070** [comment]
I am getting the same error message. Were you to resolve it?
